src/seraphis_wallet/jamtis_keys.h

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2022, The Monero Project
+// 
+// All rights reserved.
+// 
+// Redistribution and use in source and binary forms, with or without modification, are
+// permitted provided that the following conditions are met:
+// 
+// 1. Redistributions of source code must retain the above copyright notice, this list of
+//    conditions and the following disclaimer.
+// 
+// 2. Redistributions in binary form must reproduce the above copyright notice, this list
+//    of conditions and the following disclaimer in the documentation and/or other
+//    materials provided with the distribution.
+// 
+// 3. Neither the name of the copyright holder nor the names of its contributors may be
+//    used to endorse or promote products derived from this software without specific
+//    prior written permission.
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+// THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+// NOT FOR PRODUCTION
+
+////
+// Mock jamtis keys
+//
+// reference: https://gist.github.com/tevador/50160d160d24cfc6c52ae02eb3d17024
+///
+
+#pragma once
+
+//local headers
+#include "crypto/blake256.h"
+#include "crypto/chacha.h"
+#include "crypto/crypto.h"
+#include "crypto/x25519.h"
+#include "ringct/rctTypes.h"
+#include "seraphis_core/jamtis_destination.h"
+#include "serialization/keyvalue_serialization.h"
+#include "serialization/serialization.h"
+#include "wipeable_string.h"
+
+//third party headers
+
+//standard headers
+#include <functional>
+#include <vector>
+
+//forward declarations
+
+
+namespace sp
+{
+namespace jamtis
+{
+
+////
+// A set of jamtis keys for mock-ups/unit testing
+///
+struct jamtis_keys
```

Comment:
I wonder why @UkoeHB did not follow his own naming convention here; maybe he wrote the code before settling on it, because it's already `MockLedgerContext` instead of `mock_ledger_context` just 1 file further on from those `jamtis_mock_keys`. Anyway, I would vote for `JamtisKeys` in the interest of consistency.

Reply:
it was tevador :)